sdk/apimanagement/api-management-custom-widgets-tools/review/api-management-custom-widgets-tools.api.md

```diff
@@ -0,0 +1,90 @@
+## API Report File for "@azure/api-management-custom-widgets-tools"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export const ASK_FOR_SECRETS_MESSAGE_KEY = "askForSecretsMSAPIM";
+
+// @public
+export const askForSecrets: (targetModule: TTargetModule, { origin: targetOrigin, instanceId, environment }: TPortalData) => Promise<TSecrets>;
+
+// @public
+export function buildBlobConfigSrc(name: string): string;
+
+// @public
+export function buildBlobDataSrc(name: string): string;
+
+// @public
+export function buildOnChange<TValues extends TValuesCommon>(valuesDefault: TValues): TOnChange<TValues>;
+
+// @public
+export const CONFIG_FILE_NAME = "config.msapim.json";
```

Comment:
We should prefix exposed constants for disambiguation with potential collisions, maybe APIM_?